### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by knsmith**
Update path and install command.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,8 +23,8 @@ If you'd like to contribute to Superstaq, below are the instructions for install
   git clone git@github.com:Infleqtion/client-superstaq.git
   python3 -m venv venv_superstaq
   source venv_superstaq/bin/activate
-  cd superstaq-client/qiskit-superstaq
-  pip install -e ."[dev]"
+  cd  client-superstaq/qiskit-superstaq
+  python3 -m pip install -e ."[dev]"
```
Comment (richrines1) [inline README.md:27]: nits (second predates the pr)
```suggestion
  cd client-superstaq/qiskit-superstaq
  python3 -m pip install -e ".[dev]"
```
(ditto below)

File: README.md [hunk 2]
```diff
@@ -35,8 +35,8 @@ If you'd like to contribute to Superstaq, below are the instructions for install
   git clone git@github.com:Infleqtion/client-superstaq.git
   python3 -m venv venv_superstaq
   source venv_superstaq/bin/activate
-  cd superstaq-client/cirq-superstaq
-  pip install -e ."[dev]"
+  cd  client-superstaq/cirq-superstaq
```
Comment (knsmith) [inline README.md:38]:
```suggestion
  cd client-superstaq/cirq-superstaq

```

File: README.md [hunk 3]
```diff
@@ -35,8 +35,9 @@ If you'd like to contribute to Superstaq, below are the instructions for install
   git clone git@github.com:Infleqtion/client-superstaq.git
   python3 -m venv venv_superstaq
   source venv_superstaq/bin/activate
-  cd superstaq-client/cirq-superstaq
-  pip install -e ."[dev]"
+  cd client-superstaq/cirq-superstaq
+
+  python3 -m pip install -e ."[dev]"
```
Comment (richrines1) [inline README.md]:
```suggestion
  python3 -m pip install -e ".[dev]"
```

File: README.md [hunk 4]
```diff
@@ -59,8 +60,8 @@ If you'd like to contribute to Superstaq, below are the instructions for install
   git clone git@github.com:Infleqtion/client-superstaq.git
   python3 -m venv venv_superstaq
   source venv_superstaq/bin/activate
-  cd superstaq-client/supermarq-benchmarks
-  pip install -e ."[dev]"
+  cd  client-superstaq/supermarq-benchmarks
+  python3 -m pip install -e ."[dev]"
```
Comment (richrines1) [inline README.md]:
```suggestion
  python3 -m pip install -e ".[dev]"
```

File: README.md [hunk 5]
```diff
@@ -23,8 +23,8 @@ If you'd like to contribute to Superstaq, below are the instructions for install
   git clone git@github.com:Infleqtion/client-superstaq.git
   python3 -m venv venv_superstaq
   source venv_superstaq/bin/activate
-  cd superstaq-client/qiskit-superstaq
-  pip install -e ."[dev]"
+  cd client-superstaq/qiskit-superstaq
+  python3 -m pip install -e ."[dev]"
```
Comment (richrines1) [inline README.md]:
```suggestion
  python3 -m pip install -e ".[dev]"
```

**Review by richrines1 — APPROVED** [review]

**Pull request merged**